docker-entrypoint.sh

```diff
@@ -7,36 +7,44 @@ export CASSANDRA_DIR="${BUILD_DIR}/cassandra"
 
 jekyll --version
 
-# Make sure we have the latest commit of Cassandra trunk
-cd ${CASSANDRA_DIR}
+cd ${CASSANDRA_SITE_DIR}/src
+
+# Now make the docs for the latest version
+pushd ${CASSANDRA_DIR}
+git clean -xdff
 git checkout trunk
 git pull --rebase --prune
+popd
+make .build-doc .latest-doc-link
 
-# Now make the docs for the latest version
-cd ${CASSANDRA_SITE_DIR}/src
-make add-latest-doc
 
+# Uncomment for building docs for each of the released versions on the 3.11 branch
+#
+#versions="cassandra-3.11.5 cassandra-3.11.6 cassandra-3.11.7 cassandra-3.11.8"
```

Comment:
Please add a comment saying that doc generations only work for version 3.11.5 and above.